src/components/FormikForm/__tests__/FormikForm.test.tsx

```diff
@@ -388,7 +392,8 @@ describe('Test basic Components', () => {
       fireEvent.click(addButton!)
     })
     expect((inputSelect as any).value).toBe('customvalue') // selected value A
-    expect(container).toMatchSnapshot()
+    const popoverWrapper = findPopoverWrapperContainer()
+    expect(popoverWrapper).toMatchSnapshot()
```

Comment:
Can we avoid snapshot testing?

Reply:
+1

Reply:
Since a snapshot already existed previously, I have just let it remain but changed the parent container of snapshot. No new snapshot is introduced. But we can discuss this further in our team meeting